src/components/Card/Card.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, PropTypes } from 'react'
+import { View } from 'react-native'
+import Uranium from 'uranium'
+import { Elevation, Breakpoints, Colors, connectTheme } from '../../index'
+
+class Card extends Component {
```

Comment:
You can add a comment above this to automatically get the docs into carbon-ui.com, check other components for example